### ISSUE TITLE
Add rule to prevent adding argType annotations matching actions.argTypesRegex

### ISSUE THREAD

**Issue by DamienCassou** [storybookjs/eslint-plugin-storybook#83]
When the user specifies a value for `actions.argTypesRegex` such as this one (copy/pasted from the [actions addon documentation](https://storybook.js.org/docs/react/essentials/actions)):

```js
// .storybook/preview.js

export const parameters = {
  actions: { argTypesRegex: '^on.*' }
}
```

then there is no need to also specify actions matching the regular expression:

```js
export default {
  title: 'Button',
  component: Button,
  argTypes: {
    onClick: { action: 'onClick' } // unnecessary because it matches `argTypesRegex`
  }
};
```

What about adding a rule to detect and fix such cases?

Should the rule contain a configuration argument to flag/ignore an action whose name doesn't match the parameter name such as in:

```js
export default {
  title: 'Button',
  component: Button,
  argTypes: {
    onClick: { action: 'clicked' } // clicked ≠ onClick
  }
};
```
